index.js

```diff
@@ -282,6 +284,7 @@ exports.collection = function (name, opts) {
     // NOTICE: Smart Collection definition case
     opts.name = name;
     opts.isVirtual = true;
+    opts.isSearchable = opts.isSearchable ? true :  false;
```

Comment:
`opts.isSearchable = opts.isSearchable ? true : false;`

can be simplified into 

`opts.isSearchable = !!opts.isSearchable;`